tox.ini

```diff
@@ -1,14 +1,16 @@
 [tox]
-envlist = py35-django{22}
+envlist = py{35,38}-django{22,30}
 skipsdist = true
 
 [pytest]
-DJANGO_SETTINGS_MODULE = designer.settings.test
+django_settings_module = designer.settings.test
```

Comment:
Why did this change to lower case?  The pytest-django docs still have it as upper case: https://pytest-django.readthedocs.io/en/latest/#quick-start

Reply:
I guess, the script changed it to lowercase. I'll change it back to original.